catalog/ui/search-ui/standard/src/main/webapp/js/model/NearbyLocation.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+
+define([
+        'backbone',
+        'wellknown'
+       ],
+
+       function (Backbone, wellknown) {
+            "use strict";
+
+            var NearbyLocation = {};
+
+            NearbyLocation = Backbone.Model.extend({
+
+                defaults: {
+                    geo: ""
+                },
+
+                url: 'services/REST/v1/Locations/nearby/cities',
+
+                initialize: function(attributes) {
+                    console.log(attributes);
+                    if (attributes.geo) {
+                        console.log("initing new NearbyLocation object with: ");
+                        console.log(attributes.geo.toJSON());
```

Comment:
Can we remove all of these console.log messages? Console is really only for developer use and should not appear in production code.